Tasks/PowerShellV2/task.json

```diff
@@ -97,6 +97,14 @@
             "helpMarkDown": "If this is true, this task will fail if any errors are written to the error pipeline, or if any data is written to the Standard Error stream. Otherwise the task will rely on the exit code to determine failure.",
             "groupName": "advanced"
         },
+        {
+            "name": "showWarnings",
+            "type": "boolean",
+            "label": "Show warnings as azure devops warnings",
+            "required": true,
```

Comment:
We should definitely not introduce a required input - that would be a breaking change since no pipeline would have that input.

I'd prefer this to be optional and off by default